src/variables.ts

```diff
@@ -38,16 +39,23 @@ let hackerScore = 0;
 
 function incrementBy1() {
   // Increment hackerScore by 1 👇🏻
+  // hackerScore++;
+  // console.log(hackerScore)
```

Comment:
Commented out code (also called "dead code") should be removed from the file before committing